### DISCUSSION TITLE
Roblox

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#45] [General]
# Roblox

Check if Roblox is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/e6599f3f-e97c-4223-bd17-f40439a437b5

**Comment by jacksonattwood** [comment]
Runs fine on Snapdragon X Elite, 32GB RAM Surface Laptop 7, however is emulated x64

**Reply by OpenSource03** [reply]
Thanks, I was able to get it working too. I've updated the status.
